policyengine_us/parameters/gov/states/va/tax/income/deductions/itemized/limitation/itemized_deduction_rate.yaml

```diff
@@ -0,0 +1,11 @@
+description: Virginia limits the itemized deduction at this rate of the amount of the itemized deductions otherwise allowable for such taxable year.
```

Comment:
```suggestion
description: Virginia limits the itemized deductions to this rate of the amount of the itemized deductions otherwise allowable for such taxable year.
```